packages/components/src/SubHeaderBar/InputTitleSubHeader/TitleSubHeader.component.js

```diff
@@ -5,7 +5,12 @@ import { Action } from '../../Actions';
 import theme from './InputTitleSubHeader.scss';
 import { getDefaultTranslate } from '../../translate';
 
-function TitleSubHeader({ title, subTitle, onEdit, t }) {
+function noop() {}
+
+function TitleSubHeader({ title, subTitle, onEdit, editable, t }) {
+	const wordingClassNames = onEdit
+	? classNames(theme['tc-subheader-details-text-title-wording'], 'tc-subheader-details-text-title-wording')
+	: classNames(theme['tc-subheader-details-text-title-wording-with-hover'], 'tc-subheader-details-text-title-wording-with-hover');
```

Comment:
use `classNames` features instead of custom ternary
```
const wordingClassNames = classNames(
    {
        theme['tc-subheader-details-text-title-wording']: onEdit,
        'tc-subheader-details-text-title-wording': onEdit,

        theme['tc-subheader-details-text-title-wording-with-hover']: !onEdit,
        'tc-subheader-details-text-title-wording-with-hover': !onEdit,
    }
);
```

or make the ternary on the string, not entire classname

 ```
const wcn = onEdit ? 'tc-subheader-details-text-title-wording' : 'tc-subheader-details-text-title-wording-with-hover';
const wordingClassNames = classNames(theme[wcn], wcn);
```